src/index.js

```diff
@@ -1,22 +1,78 @@
 import './style.css';
-import Store from './modules/store.js';
-import UI from './modules/ui.js';
-import User from './modules/user.js';
 
-document.addEventListener('DOMContentLoaded', UI.displayUsers);
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/j22uKXxrSkg52SSFn2tq/scores';
 
+const list = document.querySelector('.list');
 const form = document.querySelector('#user-form');
+const refresh = document.querySelector('.refresh-btn');
+const getScores = async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+};
+
+const display = async () => {
+  const data = await getScores();
```

Comment:
[OPTIONAL]

- I think it would be better if you sort the scores in descending order such that the highest score stays at the top while the lowest score move to the bottom. This way, the app will be a perfect leaderboard app 😄

Reply:
Thanks! Suggestion noticed!